align/pnr/checkers.py

```diff
@@ -304,6 +304,16 @@ def addt( obj, con, tag=None):
     d['bbox'] = cnv.bbox.toList()
     d['terminals'] = new_d['terminals']
 
+    if toplevel:
```

Comment:
The other net that can appear at this level are net internal to submodules.
Maybe you can tell me what you are trying to do here.